ray-operator/apis/ray/v1alpha1/raycluster_types.go

```diff
@@ -147,6 +147,12 @@ const (
 // RayCluster is the Schema for the RayClusters API
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+//+kubebuilder:printcolumn:name="desired workers",type=integer,JSONPath=".status.desiredWorkerReplicas",priority=0
+//+kubebuilder:printcolumn:name="available workers",type=integer,JSONPath=".status.availableWorkerReplicas",priority=0
```

Comment:
I wanted to show something like Deployments' READY column with an available/desired in one column. But [only simple field access is supported](https://kubernetes.slack.com/archives/CAR30FCJZ/p1621540795101900?thread_ts=1621531572.100100&cid=CAR30FCJZ). Lmk if there's some way.

```
kubectl get deployments
NAME             READY   UP-TO-DATE   AVAILABLE   AGE
my-deployment    1/1     1            1           515d
```

Reply:
Neither do I. separate columns sounds good.